part-spec/power.json

```diff
@@ -0,0 +1,610 @@
+                "vinMax": {
+                    "description": "maximum input voltage under which the part can be expected to operate properly",
+                    "comment": "units of volts",
+                    "type": "number"
+                },
+                "dropoutVoltage": {
+                    "description": "nominal dropout voltage for ldo",
+                    "comment": "units of volts",
+                    "type": "number"
+                },
+                "ilim": {
+                    "description": "maximum sustained current output current under which the part will operate properly",
+                    "comment": "units of amps",
+                    "type": "number"
+                },
+                "turnOnTime": {
+                    "description": "time for output voltage to go from 10% vout nominal to 90% vout nominal",
+                    "comment": "units of seconds",
+                    "type": "number"
+                },
+                "rohs": {
+                    "description": "whether a part is rohs compliant",
+                    "type": "boolean"
+                },
+                "automotive": {
+                    "description": "whether a part is automotive qualified",
+                    "type": "boolean"
+                },
+                "pins": {
+                    "description": "array of pin objects with associated properties",
+                    "type": "array",
+                    "items": {
+                        "$ref": "definitions.json#/pinSpec"
+                    }
+                }
+            }
+        },
+        {
+            "title": "voltage regulator",
+            "type": "object",
+            "required": [
+                "manufacturer",
+                "mpn",
+                "datasheetVersion",
+                "type"
+            ],
+            "properties": {
+                "partType": {
+                    "description": "part type",
+                    "type": "string",
+                    "enum": [
+                        "voltageRegulator"
+                    ]
+                },
+                "manufacturer": {
+                    "description": "company that manufacturers the part",
+                    "type": "string"
+                },
+                "mpn": {
+                    "description": "manufacturer part number",
+                    "type": "string"
+                },
+                "datasheetVersion": {
+                    "description": "datasheet version encoded in this document",
+                    "type": "string"
+                },
+                "eolDate": {
+                    "description": "date this part will no longer be supported by manufacturer",
+                    "type": "string",
+                    "format": "date"
+                },
+                "type": {
+                    "description": "switching voltage regulator type",
+                    "enum": [
```

Comment:
I'd go example over enum (especially since one option here is other - that should maybe be an etc.)